docusaurus/docs/iOS/03-guides/01-client-auth.mdx

```diff
@@ -39,17 +38,9 @@ let streamVideo = StreamVideo(
 )
 ```
 
-Note the `fetchToken` method. In this method, you will need to provide your own implementation, that will load a new token for the current user. As soon as you provide the token in the `result`, the SDK will be notified about the new token and retry the failed request.
-
-When the object is created, you should connect the user to our backend. This can be done whenever you plan on using the video features.
-
-```swift
-private func connectUser() {
-    Task {
-        try await streamVideo?.connect()
-    }
-}
-```
+:::note
+The `fetchToken` method you will need to provide your own implementation, that will load a new token for the current user. As soon as you provide the token in the `result`, the SDK will be notified about the new token and retry the failed request.
```

Comment:
No strong opinion here, but I did not get this after reading it the first time. How does this sound? 🤔
```suggestion
You will need the `fetchToken` method to provide your own implementation, that will load a new token for the current user. As soon as you provide the token in the `result`, the SDK will be notified about the new token and retry the failed request.
```